arbnode/sequencer.go

```diff
@@ -265,11 +277,7 @@ func (s *Sequencer) sequenceTransactions(ctx context.Context) {
 		}
 		// try to add back to queue otherwise
 		for _, item := range queueItems {
-			select {
-			case s.txQueue <- item:
-			default:
-				item.resultChan <- errors.New("queue full")
-			}
+			s.requeueOrFail(item, ErrNoSequencer)
```

Comment:
here we're handling ErrRetrySequencer, which should mean that this sequencer just stopped being the main one. Another behavior change which I think gives a better error.